examples/openshift-lightspeed-tls.yaml

```diff
@@ -100,6 +100,16 @@ spec:
             mountPath: /app-root/config/openai
           - name: tls-certs
             mountPath: /app-root/certs
+      - name: user-data-collector-sidecar
+        image: quay.io/openshift/lightspeed-service-api-udc:latest
```

Comment:
should not be a separate/different image pullspec.

Reply:
Yes, there is a todo to just change entrypoint/command. The image should be the main ols.